wizard/src/utils/file_generation.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Mapping } from "../types";
+
+/**
+ * Generates the cleaned csv given the original csv and the correct mappings
+ * @param csv
+ * @param mappings
+ * @returns 
+ */
+export function generateCleanedCSV(csv: string[][], mappings: Mapping): string {
```

Comment:
CsvData?

Reply:
Ah, I was imagining CsvData will have the original csv (aka 2-d array), as well as any parsed stuff, so we have one object we pass around everywhere.

Reply:
ahh, ok, that makes sense, updated!